prov/verbs/src/fi_verbs.h

```diff
@@ -109,20 +113,13 @@
 extern struct fi_provider fi_ibv_prov;
 extern struct util_prov fi_ibv_util_prov;
 
-extern size_t verbs_default_tx_size;
-extern size_t verbs_default_rx_size;
-extern size_t verbs_default_tx_iov_limit;
-extern size_t verbs_default_rx_iov_limit;
-extern size_t verbs_default_inline_size;
-
-extern size_t verbs_min_rnr_timer;
-
 extern struct fi_ibv_gl_data {
 	size_t	def_tx_size;
 	size_t	def_rx_size;
 	size_t	def_tx_iov_limit;
 	size_t	def_rx_iov_limit;
 	size_t	def_inline_size;
+	size_t	min_rnr_timer;
```

Comment:
Change looks better matched with the prior commit

Reply:
Done